docs/docs/userguide/configuration/BITBUCKET_CONFIGURATION.md

```diff
@@ -194,4 +192,17 @@ bitbucket_db_master_username = "<DB_MASTER_USERNAME>"   # e.g. "postgres"
 
 ```terraform
 bitbucket_db_master_password = "<DB_MASTER_PASSWORD>"   # default value is null
-```
\ No newline at end of file
+```
+
+### Shared home snapshot id
+To restore shared home dataset, you can provide an EBS snapshot that contains content of the shared home volume.
+This volume will then be mounted to the NFS server and used when the product is started.
+
+`bitbucket_shared_home_snapshot_id` sets the id of shared home EBS snapshot.
```

Comment:
To stay consistent with: https://github.com/atlassian-labs/data-center-terraform/pull/207


```suggestion
`bitbucket_shared_home_snapshot_id` sets the id of the shared home EBS snapshot to use. This will spin up an EBS volume that will be mounted to the NFS server and used when the product is started.
```